CHANGES.rst

```diff
@@ -1,3 +1,20 @@
+20.2.4 (October 01, 2021)
```

Comment:
Took longer to get to this point than I thought. Should probably wait until Monday to release.

```suggestion
20.2.4 (October 04, 2021)
```